src/styles/pagination.scss

```diff
@@ -98,12 +108,18 @@ $block: #{$fd-namespace}-pagination;
     }
   }
 
+  &--short {
+    .#{$block}__input {
+      display: none;
+    }
+  }
+
   &--mobile {
     @include fd-pagination-mobile();
   }
 }
 
-@media (max-width: 599px) {
+@media (max-width: 1024px) {
```

Comment:
we have media queries in the mixin file. If it's missing you can add it there so we reuse it in future.